exercises/matching-brackets/README.md

```diff
@@ -0,0 +1,5 @@
+# Matching Brackets
```

Comment:
To keep a standard format and make maintenance in the future easier, the README should be generated using the configlet. You can find instructions on how to do that here: https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md#generating-a-readme

Let me know if you run into an issues or have any questions about it